spec/system/admins_spec.rb

```diff
@@ -26,4 +26,42 @@
       end
     end
   end
+
+  describe '管理者作成' do
+    let!(:admin) { create(:admin, email: 'admin@example.com', password: 'passwordpassword', password_confirmation: 'passwordpassword') }
```

Comment:
ログインに使うだけなので email, password 等は factory 任せでよいです。let, let! は好み
```suggestion
    let(:admin) { create(:admin) }
```

Reply:
[429eed3](https://github.com/kk0000-kk/sakura-market/pull/21/commits/429eed391f5b31436c72048d4d41b49ecefca3d3)